src/main/java/com/iota/iri/service/dto/RemoveNeighborsResponse.java

```diff
@@ -1,5 +1,12 @@
 package com.iota.iri.service.dto;
 
+/**
+ * This class represents the core API request 'removeNeighbors'.
+ * Removes a list of neighbors to your node. 
+ * This is only temporary, and if you have your neighbors added via the command line, they will be retained after you restart your node.
+ * The URI (Unique Resource Identification) for removing neighbors is:
+ * <b>udp://IPADDRESS:PORT</b>
+ **/
```

Comment:
Violates DRY.
you just need to write that it counts removed neighbors

Reply:
It will only be temporary if you add the neighbors again via the command line no?